envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/ControlPlane.kt

```diff
@@ -280,6 +281,42 @@ class ControlPlane private constructor(
             override fun newThread(r: Runnable) = Thread(r, "$threadNamePrefix-${counter.getAndIncrement()}")
         }
 
+        private fun meterThreadPoolExecutor(
```

Comment:
For consistency, please rename it add `new` prefix just like in the other ones.

Reply:
done